samples/hybrid-connections/java/simple-websocket-demo/README.md

```diff
@@ -0,0 +1,90 @@
+# 'Simple' Web Socket Hybrid Connection Sample 
+
+First, as described in the description [Here](https://github.com/bainian12345/azure-relay/tree/java-samples/samples/hybrid-connections/java), you will need to retrieve different parameters from the enviroment variable `RELAY_CONNECTION_STRING` for connection and authentication purposes:
+
+```	java
+	static final String CONNECTION_STRING_ENV_VARIABLE_NAME = "RELAY_CONNECTION_STRING";
+	static final Map<String, String> connectionParams = HybridConnectionUtil.parseConnectionString(System.getenv(CONNECTION_STRING_ENV_VARIABLE_NAME));
+	static final String RELAY_NAMESPACE = connectionParams.get("Endpoint");
+	static final String ENTITY_PATH = connectionParams.get("EntityPath");
+	static final String KEY_NAME = connectionParams.get("SharedAccessKeyName");
+	static final String KEY = connectionParams.get("SharedAccessKey");
+```
+
+For Hybrid Connection with authorization required, `TokenProvider` will be required to authenticate connections made to Azure Relay:
+
+```java
+TokenProvider tokenProvider = TokenProvider.createSharedAccessSignatureTokenProvider(KEY_NAME, KEY);
+```
+
+## Listener 
```

Comment:
>Listener [](start = 3, length = 8)

Can you model this listener section on the C# version of the same? (No I didn't write that)
[https://github.com/Azure/azure-relay/tree/master/samples/hybrid-connections/dotnet/simple-websocket#server](https://github.com/Azure/azure-relay/tree/master/samples/hybrid-connections/dotnet/simple-websocket#server)